src/plugins/internal/pluginsconfiguration.cpp

```diff
@@ -76,27 +79,78 @@ mu::async::Channel<mu::io::path> PluginsConfiguration::userPluginsPathChanged()
     return m_userPluginsPathChanged;
 }
 
-mu::ValCh<CodeKeyList> PluginsConfiguration::installedPlugins() const
+IPluginsConfiguration::ConfiguredPluginHash PluginsConfiguration::configuredPlugins() const
```

Comment:
Please, add TRACEFUNC here

Reply:
Isn't this method called too often? It reads the plugins file and has no cache, so it may cause performance problems